vocabularies/Org.OData.Capabilities.V1.xml

```diff
@@ -661,6 +679,112 @@ supported:
         </Property>
       </ComplexType>
 
+      <Term Name="RetrieveRestrictions" Type="Capabilities.RetrieveRestrictionsType" AppliesTo="EntitySet Singleton FunctionImport">

```

Comment:
I suggest 'ReadRestrictions' since the other restrictions are named 'update', 'insert' and 'delete' which match with the CRUD naming.

Reply:
The R in CRUD is (at least at SAP) mostly interpreted as "read single" or "get by key", whereas "get list" is referred to as Q, leading to CRUDQ. Which also matches OpenAPI where "get by key" and "get list" are different path items with different sets of parameters (headers and query options), different descriptions, ....

"Retrieve" was chosen to avoid that possible confusion and to mean "R and Q".

Reply:
Changed to ReadRestrictions as decided in TC meeting on 2019-02-14, also changed property Retrievable to Readable.

Descriptions still use "retrieve" to match wording of Part 1: Protocol